src/main/java/io/cdap/e2e/utils/SeleniumHelper.java

```diff
@@ -131,4 +132,22 @@ public static boolean verifyElementPresent(String locator) {
       return false;
     }
   }
+
+  public static boolean waitForParticularTime(int time) throws InterruptedException {
+    Thread.sleep(time);
+    return true;
+  }
+
+  public static String readErrors(String property) throws IOException {
+    Properties prop = new Properties();
+    InputStream input = null;
+    try {
+      input = ClassLoader.getSystemClassLoader().getResourceAsStream("ErrorMessage.properties");
```

Comment:
It's usually better to use `this.getClass().getResourceAsStream("/...")`

Reply:
This method is static and getting utilised everywhere. this. cannot be used in non static method.

Reply:
It's still generally better to use specific class classloader. In complex environments system classloaded may not have the classes needed. E.g. in static context you can use `SeleniumHelper.class.getResourceAsStream("/...")`

Reply:
changes made for the same